interface/billing/billing_report_xml.php

```diff
@@ -1,78 +0,0 @@
-<?php 
```

Comment:
I think can go (along with the upload.inc and need_service.inc).
Any reasons to keep these @sunsetsystems ?